ldap/servers/slapd/ssl.c

```diff
@@ -966,9 +966,12 @@ check_private_certdir()
 
     if (!tmp_private) {
         /* tmp is not a private name space */
-        slapi_log_err(SLAPI_LOG_WARNING, "Security Initialization",
+        if (_security_library_initialized != 0) {
```

Comment:
Is not the test inverted ?  
 The comment says the first time but (_security_library_initialized != 0) means every time except the first! ...

Reply:
Indeed, and in my second test I misread the invalid output as the correct and vice versa. I feel quite silly now :(